src/index.js

```diff
@@ -71,6 +74,18 @@ async function main() {
   const mangataAddress = alice.assets[1].address;
   const turingAddress = alice.assets[2].address;
 
+  const lquidityToken = 'MGR-TUR';
+  console.log(`Checking how much reward available in ${lquidityToken} pool ...`);
+  const rewardAmount = await mangataHelper.calculateRewardsAmount(mangataAddress, lquidityToken);
+  console.log(`Claimable reward in ${lquidityToken}: `, rewardAmount);
+
+  const liquidityBalance = await mangataHelper.getBalance("MGR-TUR", mangataAddress);
+  // const free = liquidityBalance.free.div(new BN('1000000000000000000'));
```

Comment:
free balance of MGR-TUR never changes. Only reserved MGR-TUR increases after XCM.